docs/articles/documentation/extending-testcafe/custom-reporter/reporter-methods.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: docs
+title: Reporter Methods
+permalink: /documentation/extending-testcafe/custom-reporter-plugin/reporter-methods.html
+checked: false
+---
+# Reporter Methods
+
+To create your [custom reporter](index.md#implementing-the-reporter), you need to implement the following reporter methods.
+
+> You can use [helper methods](helper-methods.md) within the reporter methods to output required data.
+
+## reportTaskStart
+
+Fires when a test task starts, which happens at the beginning of testing.
+
+```text
+reportTaskStart (startTime, userAgents, testCount)
+```
+
+Parameter    | Type             | Description
+------------ | ---------------- | ---------------------------------------
+`startTime`  | Date             | The date and time when testing started.
+`userAgents` | Array of Strings | The list of browsers used for testing.
+`testCount`  | Number           | The total number of tests to run.
+
+**Example**
+
+```js
+import moment from 'moment';
+
+reportTaskStart (startTime, userAgents, testCount) {
+    this.startTime = startTime;
+    this.testCount = testCount;
+
+    const time = moment(startTime).format('M/D/YYYY h:mm:ss a');
```

Comment:
Moment is available via `this`: https://github.com/MargaritaLoseva/testcafe/blob/4b27df2b5251f27b90d61fd6e68acb22520b1f8d/docs/articles/documentation/extending-testcafe/custom-reporter/helper-methods.md#moment

BTW, I guess we should rename `Helper methods` topic to `Helpers` since it contains not only helpers